packages/core/src/routes/well-known.ts

```diff
@@ -24,17 +24,21 @@ export default function wellKnownRoutes<T extends AnonymousRouter>(
   } = queries;
 
   if (tenantId === adminTenantId) {
-    router.get('/.well-known/endpoints/:tenantId', async (ctx, next) => {
-      if (!ctx.params.tenantId) {
-        throw new RequestError('request.invalid_input');
-      }
+    router.get(
+      '/.well-known/endpoints/:tenantId',
+      koaGuard({ response: z.object({ user: z.string().url() }), status: 200 }),
+      async (ctx, next) => {
+        if (!ctx.params.tenantId) {
+          throw new RequestError('request.invalid_input');
+        }
 
-      ctx.body = {
-        user: getTenantEndpoint(ctx.params.tenantId, EnvSet.values),
-      };
+        ctx.body = {
+          user: getTenantEndpoint(ctx.params.tenantId, EnvSet.values).toString(),
```

Comment:
Should we keep returning an URL-typed `user`? The change on the response type is regarded as BREAKING CHANGE?

Reply:
Maybe add a comment so that reviewers can better understand this change ❤️